app/app.module.ts

```diff
@@ -0,0 +1,16 @@
+import { DoBootstrap, NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { UpgradeModule } from '@angular/upgrade/static';
+
+@NgModule({
+  imports: [
+    BrowserModule,
+    UpgradeModule,
+  ],
+})
+export class AppModule implements DoBootstrap {
+  constructor(private upgrade: UpgradeModule) { }
```

Comment:
Lets write it like this 
`constructor(@Inject(UpgradeModule) private upgrade: UpgradeModule) {}`

Reply:
Fixed. It is working now. Thanks.